src/main/java/com/timelogr/enterprise/service/ITimeLogrService.java

```diff
@@ -0,0 +1,9 @@
+package com.timelogr.enterprise.service;
+
+import com.timelogr.enterprise.dto.Employee;
+import java.util.List;
+
+public interface ITimeLogrService {
+
+    List<Employee> getAllEmployees();
+}
```

Comment:
This is the interface class that was added, this will "link" the service class with the interface and the interface with the controller/tests/any other part of your program where it is necessary.